### PR TITLE
Adjust a little bit travis and tox to test more

### PR DESCRIPTION
**Pull request by pypingou**
Signed-off-by: Pierre-Yves Chibon <pingou@pingoured.fr>

### CODE REVIEW

**Comment by pypingou** [comment]
I've added a few commits since you reviewed this PR, if you would not mind having a look at it again :)

(they should make travis happy again!)

File: nuancier/lib/model.py [hunk 1]
```diff
@@ -362,7 +362,7 @@ def get_results(cls, session, election_id):
             Candidates.date_created,
             Candidates.date_updated
         ).order_by(
-            'votes DESC'
+            sa.text('votes DESC')
```
Comment (Zlopez) [inline nuancier/lib/model.py]: Is this really needed or it is just a good habit?

File: tests/test_nuancier.py [hunk 2]
```diff
@@ -1849,6 +1849,7 @@ def test_update_candidate(self):
                 in output.data.decode('utf-8'))
 
         upload_path = os.path.join(PICTURE_FOLDER, 'F21')
+        print(upload_path)
```
Comment (Zlopez) [inline tests/test_nuancier.py]: You probably forgot to delete this print.

File: nuancier/lib/model.py (hunk 1 above)
Comment (pypingou) [reply nuancier/lib/model.py]: it seems sqlalchemy is now making this error (it was one of the main reason travis was failing)

File: tests/test_nuancier.py (hunk 2 above)
Comment (pypingou) [reply tests/test_nuancier.py]: I did, thanks :)

**Comment by pypingou** [comment]
updated :)

**Comment by pypingou** [comment]
Let's wait for travis to get back to us, but thanks for your review :)

**Comment by Zlopez** [comment]
No problem, I was just in the middle of going through http://ambre.pingoured.fr/fedora-infra/ BTW Really nice page @pypingou, thank for it

**Comment by pypingou** [comment]
Thanks!

**Pull request merged**